### PR TITLE
Pwatson updated folder struct

### PR DESCRIPTION
**Pull request by pwatsonr**

### CODE REVIEW

File: service/idea_service.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from repository.orm.Idea import Idea
+
+
+class IdeaService:
+    def __init__(self):
+        return
+
+    def get_all(self):
```
Comment (pwatsonr) [inline service/idea_service.py:8]: @adamaslan I would recommend you take a look at paginate so it does not return  1000s of records you can set a default.

https://flask-sqlalchemy.palletsprojects.com/en/2.x/api/ 
https://stackoverflow.com/questions/9916094/sqlalchemy-and-going-through-a-large-result-set

Comment (adamaslan) [reply service/idea_service.py:8]: Oh very cool...looking to use Axios on the frontend - working on this with Trisha who ordered massive rugs from Wayfarer

**Comment by pwatsonr** [comment]
@adamaslan a cool idea to add to this idea system is Kafka, so when upserts/deletes happened you don't need to write a loop to hit this service to check for updates. You can just use Kafka for javascript to subscribe to the topic and will automatically update your pages when new messages are consumed. 

Python Kafka -> https://kafka-python.readthedocs.io/en/master/
Js Kafka -> https://kafka.js.org/

You could also use this to tie multiple services together if you like

**Comment by adamaslan** [comment]
> 

Ohhh very coool will check it out getting very into backend things these days...looking at Node and express as well mongodb with graphql and apollo

**Pull request merged**